_data/blogs.yml

```diff
@@ -191,4 +191,9 @@
 - url: https://saarthakchaturvedi05.wordpress.com/2017/12/24/interview-with-long-term-open-source-contributors/
   title: Interview with Two Long-Term Open Source Contributors
   img: saarthakchaturvediinterview.jpg
-  author_name: Saarthak Chaturvedi
\ No newline at end of file
+  author_name: Saarthak Chaturvedi
+  
+- url    : https://amantiwari123.wordpress.com/2017/12/27/my-journey-to-google-code-in-gci-2017-18/
+  title  : My Journey to Google Code In ( GCI 2017-18).
+  img    : https://amantiwari123.files.wordpress.com/2017/12/img_20171227_140128.jpg?w=1272
```

Comment:
You don't need to provide a link here. Instead add an image of size 240*240px in the `img/blog/` folder and write the filename in the value for `img` key.